ykman/cli/info.py

```diff
@@ -64,6 +80,23 @@ def info(ctx):
     if config.configuration_locked:
         click.echo('Configured applications are protected by a lock code.')
     click.echo()
+
+    if dev.is_fips:
+        click.echo('This YubiKey is capable of FIPS mode.')
+
+        fips_status = get_overall_fips_status(dev.serial)
+
+        click.echo('FIPS mode active: {}'.format(
+            'Yes' if all(fips_status.values()) else 'No'))
+
+        status_keys = list(fips_status.keys())
+        status_keys.sort()
+        for status_key in status_keys:
+            click.echo('  - {}: {}'.format(
+                status_key, 'Yes' if fips_status[status_key] else 'No'))
+
+    click.echo()
```

Comment:
This one is just need if fips, leads to extra blank line on regular devices.

Reply:
Good catch, thanks!